### PR TITLE
✏️ open url of etherscan

### PR DESCRIPTION
**Pull request by jasonzhouu**
Fixes: #
the param of _viewOnEtherscan is url of tx in etherscan, instead of txid.

Explanation:  
open url of tx in etherscan

### CODE REVIEW

**Comment by Gudahtt** [comment]
Thanks for the contribution! Could you run `yarn lint:fix` to fix the lint errors?

**Comment by jasonzhouu** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by jasonzhouu** [comment]
@Gudahtt Thank you for reply! fixed.

File: yarn.lock [hunk 1]
```diff
@@ -7361,9 +7361,9 @@ camelcase@^6.0.0:
   integrity sha512-8KMDF1Vz2gzOq54ONPJS65IvTUaB1cHJ2DMM7MbPmLZljDH1qpzzLsWdiN9pHh6qvkRVDTi/07+eNGch/oLU4w==
 
 caniuse-lite@^1.0.30000810, caniuse-lite@^1.0.30000844, caniuse-lite@^1.0.30001035, caniuse-lite@^1.0.30001097, caniuse-lite@^1.0.30001157:
-  version "1.0.30001162"
-  resolved "https://registry.yarnpkg.com/caniuse-lite/-/caniuse-lite-1.0.30001162.tgz#9f83aad1f42539ce9aab58bb177598f2f8e22ec6"
-  integrity sha512-E9FktFxaNnp4ky3ucIGzEXLM+Knzlpuq1oN1sFAU0KeayygabGTmOsndpo8QrL4D9pcThlf4D2pUKaDxPCUmVw==
+  version "1.0.30001241"
```
Comment (brad-decker) [inline yarn.lock:7364]: [Q] was this a necessary change? If not could we remove it and rebase?

**Review by brad-decker — CHANGES_REQUESTED** [review]
One minor thing then we can get this landed .sorry for the delay @jasonzhouu

**Comment by darkwing** [comment]
Superceded by https://github.com/MetaMask/metamask-extension/pull/12609

**Pull request closed without merging**